sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Framework/AuthenticationEventMetadata.cs

```diff
@@ -47,7 +47,7 @@ internal AuthenticationEventRequestBase CreateEventRequest(HttpRequestMessage re
         {
             AuthenticationEventRequestBase eventRequest = (AuthenticationEventRequestBase)Activator.CreateInstance(RequestType, new object[] { request });
             PropertyInfo responseInfo = eventRequest.GetType().GetProperty("Response");
-            PropertyInfo dataInfo = eventRequest.GetType().GetProperty("Payload");
+            PropertyInfo dataInfo = eventRequest.GetType().GetProperty("Data");
```

Comment:
Would it be possible to use `nameof` here for the magic strings?

Reply:
Won't fix.